<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceManager</class>
 <widget class="QWidget" name="InterfaceManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>460</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>InterfaceManager</string>
  </property>
  <widget class="QLabel" name="label_interface">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>网络适配器：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="interface_list">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>360</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_dhcp">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>470</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>DHCP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_addr">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IP 地址：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_addr">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>360</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_netmask">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>子网掩码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_netmask">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>360</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_gateway">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>360</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_gateway">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>默认网关：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_dns1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>360</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_dns1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DNS 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dns2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DNS 2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_dns2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>360</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_apply_addr">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>应用</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_apply_dns">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>390</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>应用</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_refresh">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_apply_auto_dns">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>390</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>自动获取</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_apply_all">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>390</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>应用所有</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
